iOS_SDK/OneSignalSDK/UnitTests/Shadows/OneSignalLocationOverrider.m

```diff
@@ -119,7 +119,7 @@ - (void)overrideRequestWhenInUseAuthorization {
 - (void)overrideStartUpdatingLocation {
     // If iOS is less than 8.0, the startUpdatingLocation prompts the user
```

Comment:
Adjust the comment here since we dont check for less than iOS 8 anywhere

Reply:
done!